### PR TITLE
Mobile-First-Portfolio: Addition of index.html file, style.css file and  linters

### PR DESCRIPTION
**Pull request by AsciencioAlex**
I did the following in this project
1. Created an index.html file for the mobile portfolio
2. Create a style.css file that styles up the page
3. Added linters folders to the project for removing linters errors

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @AsciencioAlex, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has missing headers and sections, Kindly note that the only section which can be removed is the `FAQ` section and the `Live Demo` Section if the project has not been deployed and deployment is not a requirement. 

- [ ] Kindly add the `Getting Started`, `Author`, and `Future Features` sections to your ReadMe file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+       <link rel="stylesheet" href="style.css">
+    </head>
+ <body>   
+      <header class="navbar navbar-expand-lg fixed-top navbarScroll">
+        
+          <a class="navbar__logo" href="#"> My Logo</a>
+          <div class="navbar__label_container">
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span></div>
```
Comment (Efiamotu-1) [inline index.html]: - Your hamburger button should be wrapped around a `nav tag`. The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices. :+1: :sparkles:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <section class="navigation">    
+            <h1 class="navigation__bar">I'm Alex Asciencio. Glad to see you!</h1>
+            <main>
+              <p class="me">
+                I'm a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don't hesitate to
+                contact me.
+              </p>
+            </main>
```
Comment (Efiamotu-1) [inline index.html]: - [ ]  I think the `main tag` shouldn't be used to wrap the `paragraph`. Please follow this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) to read me about the `main` tag and how it should be used in sectioning the entire webpage :+1:

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] As shown in the figma design below you are not posititoning the content of the header properly and the spacing of the content is not the same as the provided figma design as shown by the red lines. Please i kindly suggest that you make the appropriate correction becasue it is important that your design matches completely the provided figma template as this shows how much attention to details you pay as a good frontend developer :+1: :sparkles: 
- figma design
dis
- your design
yed

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @AsciencioAlex , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <header class="navbar">
+        <div class="navbar__header">
+            <div class="navbar__header__content">
+              <div class="navbar__logo" href="#"> My Logo</div>
+                <div class="navbar__label_container">
+                  <span class="__label__list_container"></span>
+                  <span class="__label__list_container"></span>
+                  <span class="__label__list_container"></span></div>
+
+            </div>
```
Comment (tresorsawasawa) [inline index.html:21]: - [ ] As mentioned by the previous reviewer and for good semantics reasons, kindly ensure that your Hambarger is wrapped in a `nav` tag i.e. its container tag should be a `<nav>` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        </div> 
+      </header>
+  
+      <section class="navigation"> 
+        <div class="background"> 
+          <main>
```
Comment (tresorsawasawa) [inline index.html]: - [ ] As mentioned by the previous reviewer and for best semantics tag, kindly ensure that you use the `main` tag at it s expected place. Below is an example of where to use the `main` tag in a page.
```html
  <header>
    <!-- Header content goes here -->
  </header>

  <main>
    <section>
      <!-- section 1 content goes here -->
    </section>

    <section>
      <!-- section 2 content goes here -->
    </section>

    <section>
      <!-- section 3 content goes here -->
    </section>
    ...
  </main>

  <footer>
    <!-- footer content goes here -->
  </footer>
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>portfolio mobile</title>
+       <link rel="stylesheet" href="style.css">
+       <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    </head>
+ <body>   
+      <header class="navbar">
+        <div class="navbar__header">
+            <div class="navbar__header__content">
+              <div class="navbar__logo" href="#"> My Logo</div>
```
Comment (tresorsawasawa) [inline index.html:15]: - [ **OPTIONAL** ] **RECOMMENDED:** As this comment was not pointed out by the previous reviewer, kindly ensure that your logo is a link (i.e. it uses a `<a>` tag). This is for three reasons: convention, semantics, and user experience(UX).

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @AsciencioAlex 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Alfredbis29) [reply index.html:1]: - [OPTIONAL ] You are not positioning the content of the header appropriately, as shown in the figma design below, and the spacing of the content is not the same as the given figma design. compare your Design to the figma one are not really similar , kindly fix your `background-image` and all it contents  , Kindly fix this :point_down: 
_Even the word Glad to see you_ is not arranged well compare to the design :pray: 

Your Design 
de

Figma Design 
Screenshot from 2023-07-07 01-52-20

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @AsciencioAlex 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Well structured files :white_check_mark:
  * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] Check Under the Review

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**